app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt

```diff
@@ -152,13 +144,8 @@ class MainActivityTest {
         assertEquals("2", getDisplayedNumber())
         checkFormula("7%5")
 
-        setDouble(8.0)
-        handleOperation(ROOT)
-        assertEquals("16", getDisplayedNumber())
-        checkFormula("√256")
-
         activity.calc.handleClear()
-        assertEquals("1", getDisplayedNumber())
+        assertEquals("0", getDisplayedNumber())
```

Comment:
The test makes no sense. The assert expects that we return `1` as a result when clearing the calculator. I fixed the test to make it pass

Reply:
Agreed its also way too long and testing a whole bunch of crap :/